test/functionalTest/cases/1038_ngsiv2_geo_location_entities/errors_general.test

```diff
@@ -227,13 +227,13 @@ echo
 01. Create entity with one geo:point and one geo:line -> fail
 =============================================================
 HTTP/1.1 413 Request Entity Too Large
-Content-Length: 96
+Content-Length: 148
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "No more than one geo-location attribute allowed",
+    "description": "You cannot use more than one geo location attribute when creating an entity [see Orion user manual]",
```

Comment:
This (an other similar ones) is a little adaptation in description text, to align with existing messages for the same purpose. No need to mention in CNR.

NTC (just informative).